__tests__/app.test.js

```diff
@@ -71,3 +70,19 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+describe('/api/articles', () => {
+  test('GET:200 gets all articles with specified columns', () => {
+    return request(app)
+      .get('/api/articles/')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy('created_at');
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe('string');
```

Comment:
are there any other keys that the articles should have? or any that they _shouldn't_ have? we should check for those too